index.html

```diff
@@ -1 +1 @@
\ No newline at end of file
+<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><meta name="description" content="Exercise jr talent program pug and accessibility"><title>Jr-Talent-Program</title><link rel="stylesheet" href="./css/styles.css"><link rel="stylesheet" href="./css/grid.css"></head><body class="container"> <header class="header"><h1 class="header__title">Make huge moves</h1><figure><a class="header__link" href="https://www.hugeinc.com/"><img class="header__logo" src="./assets/Huge_logo_obscuro.png" alt="Huge logo"></a></figure><div class="regionalMenu" role="navigation" aria-label="Regional menu"><label class="regionalMenu__options" for="languaje">Languaje <select class="regionalMenu__select" name="languaje" id="languaje"><option aria-checked="true" value="spanish">ES</option><option aria-checked="false" value="english">EN</option></select></label><label class="regionalMenu__options" for="currency">Currency<select class="regionalMenu__select" name="currency" id="currency"><option aria-checked="true">USD</option><option aria-checked="false">COP</option></select></label></div></header><nav class="navbar"><ul><li><a class="navbar__items" href="Program">Program</a></li><li><a class="navbar__items" href="Tech">Tech</a></li><li><a class="navbar__items" href="Talent">Talent</a></li></ul></nav><main class="main"><article class="banner" role="banner"><figure><img src="./assets/program.jpg" alt=""><figcaption>Program</figcaption><img src="./assets/talent.jpg" alt=""><figcaption>Talent</figcaption><img src="./assets/tech.jpg" alt=""><figcaption>Tech</figcaption></figure></article></main><form class="contactForm"><div class="contactForm__items"> <label id="username">Names </label><input type="text" name="username" placeholder="enter your name"></div><div class="contactForm__items"> <label id="lastname">LastName</label><input type="text" name="lastname" placeholder="enter your lastname"></div><div class="contactForm__items"> <label id="email">Email</label><input type="email" name="email" placeholder="enter your email"></div><div class="contactForm__items"> <label id="message">Message</label><textarea name="message" cols="20" rows="4"> </textarea></div><div class="contactForm__submit"><button type="submit" value="Submit">Submit</button></div></form><footer class="footer"> <p class="footer__title">Powered by the M1 talent program at <a href="https://www.hugeinc.com/"><span>Huge</span></a></p></footer></body></html>
```

Comment:
This file "index.html" correct me if Im wrong, but this is auto generated by pug (you don't code here I guess) if that so, we tend to ignore this kind of files with the `.gitignore` because this will cause some noise when we push the changes to the repo reflecting several changes every time we do a change in the code, and also we can create this file again by running the `pug build` so its not critical if we don't track that.

Reply:
I'm going to investigate about it, to solve it